<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Center</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="MonitorGroup_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>System Monitor</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="PresetMode_1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Preset - Mode 01</string>
    </property>
    <widget class="QPushButton" name="Run_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Load_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clear_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PresetMode_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Preset - Mode 02</string>
    </property>
    <widget class="QPushButton" name="Run_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Load_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clear_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PresetMode_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Preset - Mode 03</string>
    </property>
    <widget class="QPushButton" name="Run_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Load_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clear_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Device_Control_1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Device Control</string>
    </property>
    <widget class="QGroupBox" name="Device_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Device 01</string>
     </property>
     <widget class="QPushButton" name="Dev1_Start_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>88</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Dev1_Stop_1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>88</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Device_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Device 02</string>
     </property>
     <widget class="QLabel" name="Dev2_Switch_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../ec0b3763/switch-off.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Device_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Device 03</string>
     </property>
     <widget class="QSlider" name="Dev3_Slider_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Notification_1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>651</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Notification</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="SystemControl_1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>System Control</string>
    </property>
    <widget class="QPushButton" name="Save_Setting_1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Setting</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clear_Chart_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Chart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Clear_Notification_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Notification</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Load_Setting_1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Setting</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Exit_1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Indicator_1">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>181</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Indicator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
